templates/bake/Template/view.twig

```diff
@@ -118,7 +118,7 @@
 {% endif %}
 {% set relations = associations.BelongsToMany|merge(associations.HasMany) %}
 {% for alias, details in relations %}
-{% set otherSingularVar = alias|variable %}
+{% set otherSingularVar = details.singular|variable %}
```

Comment:
Why is a new `singular` key needed? You can directly do `alias|singularize|variable`.

Reply:
oh.. that's cool!

Reply:
ping @arodu 
You want to finish this up for merge?

Reply:
Yes, just update the template